### DISCUSSION TITLE
MIME Types

### DISCUSSION THREAD

**Discussion by Roguelazer** [FiloSottile/age#265] [General]
Are there any plans to apply for MIME types for age-encrypted files? It always makes me a little sad to have to serve something as `application/octet-stream`...

**Comment by FiloSottile** [comment]
Oh that's a great idea! It looks like we'd have to register `application/vnd.age`, since we're not a standard published by a recognized standards-related body? (I don't even want to think about the politics associated with that.)

**Reply by spacesynth** [reply]
Registration Template:
https://datatracker.ietf.org/doc/html/rfc2048#section-2.8

**Comment by Roguelazer** [comment]
https://twitter.com/FiloSottile/status/1445472455519195140 looks like this got done!

**Reply by FiloSottile** [reply]
Yes! Thank you for suggesting that!
